src/pages/Login.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { fetchToken } from '../actions';
+
+import logo from '../trivia.png';
+
+class Login extends React.Component {
+  constructor() {
+    super();
+
+    this.handleOnChange = this.handleOnChange.bind(this);
+
+    this.state = {
+      email: '',
+      playerName: '',
+      disabled: true,
+    };
+  }
+
+  handleOnChange({ target }) {
+    const { name, value } = target;
+    this.setState(
+      {
+        [name]: value,
+      },
+      () => {
+        const { email, playerName } = this.state;
+
+        if (email.length > 0 && playerName.length > 0) {
+          this.setState({ disabled: false });
+        } else {
+          this.setState({ disabled: true });
+        }
+      },
+    );
+  }
+
+  render() {
+    const { email, playerName, disabled } = this.state;
+    // Destruturate my login action as a props
+    const { requestToken } = this.props;
+
+    return (
+      <div>
+        <div>
+          <Link to="/config">
+            <button
+              type="button"
+              data-testid="btn-settings"
+            >
+              Jogar
```

Comment:
Lugh, faz sentido mudar esse texto para 'Settings' (Olivia feelings) =D

```suggestion
              Settings
```